dhis-2/dhis-services/dhis-service-tracker/pom.xml

```diff
@@ -15,53 +15,161 @@
 
     <dependencies>
 
+        <dependency>
+            <groupId>org.hisp.dhis</groupId>
+            <artifactId>dhis-api</artifactId>
+            <exclusions>
+                <exclusion>
+                    <groupId>*</groupId>
+                    <artifactId>*</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
         <dependency>
             <groupId>org.hisp.dhis</groupId>
             <artifactId>dhis-service-schema</artifactId>
+            <exclusions>
+                <exclusion>
+                    <groupId>*</groupId>
+                    <artifactId>*</artifactId>
+                </exclusion>
+            </exclusions>
         </dependency>
         <dependency>
             <groupId>org.hisp.dhis</groupId>
             <artifactId>dhis-service-core</artifactId>
+            <exclusions>
+                <exclusion>
+                    <groupId>*</groupId>
```

Comment:
why do we need the dependency but exclude all its dependencies? shouldnt that be resolved since all of the modules have the same parent with the same dependency versions?

Reply:
Removed, this is how I really test that anything is coming from other dependencies.
Just sneaky slipped into the PR